### PR TITLE
LUI-142 : Removed required * notation for the family name

### PR DESCRIPTION
**Pull request by vinodharanie**
Ticket: https://issues.openmrs.org/browse/LUI-142

I have removed the mandatory notation ("*") for the family name since the family name is not compulsory to a patient details

### CODE REVIEW

**Comment by ODORA0** [comment]
Have you taken a look at https://wiki.openmrs.org/display/docs/Pull+Request+Tips?

**Comment by vinodharanie** [comment]
@ODORA0, can you please review my code?

**Comment by vinodharanie** [comment]
@dkayiwa, can you please do a code review?

File: omod/src/main/webapp/portlets/nameLayout.jsp [hunk 1]
```diff
@@ -164,7 +164,7 @@
 									<c:forEach items="${line}" var="token" varStatus="tokenStatus">
 										<c:if test="${token.isToken == model.layoutTemplate.layoutToken}">
 											<td><openmrs:message code="${token.displayText}" />
-											    <c:if test="${token.displayText == 'PersonName.givenName' || token.displayText == 'PersonName.familyName'}">
+											    <c:if test="${token.displayText == 'PersonName.givenName'}">
```
Comment (RishavR) [inline omod/src/main/webapp/portlets/nameLayout.jsp]: Acceptable.

File: omod/src/main/webapp/portlets/nameLayout.jsp [hunk 2]
```diff
@@ -11,7 +11,7 @@
 				<c:forEach items="${line}" var="token">
 					<c:if test="${token.isToken == model.layoutTemplate.layoutToken}">
 						<th><openmrs:message code="${token.displayText}"/>
-						    <c:if test="${token.displayText == 'PersonName.givenName' || token.displayText == 'PersonName.familyName'}">
+						    <c:if test="${token.displayText == 'PersonName.givenName'}">
```
Comment (RishavR) [inline omod/src/main/webapp/portlets/nameLayout.jsp]: Acceptable.

**Comment by vinodharanie** [comment]
@RishavR,

create-patient
patient

Please find the attachment here

**Comment by dkayiwa** [comment]
Thanks @fanyui  for approving 😊

**Pull request merged**